bayesian/api_v1.py

```diff
@@ -111,6 +114,17 @@ def liveness():
     return jsonify({}), 200
 
 
+@api_v1.route('/repr_licesne')
```

Comment:
I think you should implement this API end-point something like this https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/bayesian/api_v1.py#L490

Reply:
+1, please make use of flask-restfull when we already use it in the code.